mlir/lib/Dialect/Rock/Tuning/ConvContext.cpp

```diff
@@ -46,7 +47,7 @@ ConvolutionDims ConvolutionContext::getConvDims() {
 }
 
 ConvolutionContext mlir::rock::populateConvContext(Operation *op) {
-  ConvOpType opType = obtainConvDirection(op);
+  ConvOpType opType = *obtainConvDirection(op);
```

Comment:
I think we dont need this opType as part of the ConvContext after this change.
My quick investigation suggests that only uses of populeConvContext remains in ConvToGemm (after this PR) where the caller knows the direction.